apps/server/doc/testing.md

```diff
@@ -57,6 +60,42 @@ Your test should be structured in three seperate areas, each distinguished by at
 
 this is known as the AAA-pattern.
 
+The tests for a unit should cover as much scenarios as possible. Parameters and the combination of parameters can often take numerous values. Therefore it largely differs from case to case what a sufficient amount of scenarios would be. Parameter values that a theoretically not possible by the typescript type definition should be ignored as a test case. 
```

Comment:
The last sentence should probably be: "Parameter values that are theoretically not possible by the typescript type definition should be ignored as a test case."